packages/app/src/runner/ResizablePanels.vue

```diff
@@ -4,7 +4,6 @@
     class="flex"
     :class="{
       'select-none': panel1IsDragging || panel2IsDragging,
-      'overflow-x-hidden': isFirefox
```

Comment:
Love that we can remove browser-specific styling and just apply one thing that works everywhere. +1